### ISSUE TITLE
Change filetype defs seem to don't have any effect

### ISSUE THREAD

**Issue by frlan** [geany/geany-osx#7]
I've updated the filetype-defintions for python via the Tools->Configuration Files->Filetype Configuration->Scripting Languages->fileytypes.python and added 

> [indentation]
> width=4
> type=0

which should be also the default. But even after restarting Geany when opening a Python-file default indent is done via \t.  (1.30.1 on 10.11.6)

**Comment by elextr** [comment]
Is it being overridden by a project?

**Comment by frlan** [comment]
No, it isn't

**Comment by frlan** [comment]
At least there is no open project according to the project menu.

**Comment by techee** [comment]
@frlan I just tried and it works for me on Mac too. Are you sure you closed the file before retrying? (Otherwise the indentation type is stored into the session file.). Also don't you use indentation autodetection which could override it?

I don't think spaces are default for Python - the default is to use global or project settings.

That said, there really is a bug - the local filetype config file should be created based on the global one which doesn't happen now. I've just fixed it in

https://github.com/geany/geany/pull/1482
